Update seacas to 2025-02-27 release

cticenhour opened the pull request:
Refs #21665

cticenhour committed:
Update seacas to 2025-02-27

Refs #21665

cticenhour committed:
Update seacas package templates with versioner script

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/2719420/), step Versioner verify on e0946e9 wanted to post the following:

### Versioner templates

Found 16 templates, 0 failed


### Versioner versions

Found 10 packages, 1 changed, 0 failed

| package   | status    | hash    |   to hash | version            | to version         |
|-----------|-----------|---------|-----------|--------------------|--------------------|
| seacas    | CHANGE OK | 2dd352c |  6.7e+221 | 2024.08.15=build_2 | 2025.02.27=build_0 |

cticenhour commented:
@loganharbour Looks like the Versioner output decided to format the "to hash" for this change as a number in scientific notation. Odd!

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/2737017/), step Versioner verify on 55e2493 wanted to post the following:

### Versioner templates

Found 16 templates, 0 failed


### Versioner versions

Found 10 packages, 1 changed, 0 failed

| package   | status    | hash    |   to hash | version            | to version         |
|-----------|-----------|---------|-----------|--------------------|--------------------|
| seacas    | CHANGE OK | 2dd352c |  6.7e+221 | 2024.08.15=build_2 | 2025.02.27=build_0 |

moosebuild commented:
Job [Coverage](https://civet.inl.gov/job/2737064/), step Generate coverage on 55e2493 wanted to post the following:


## Framework coverage

<table>
  <thead>
    <tr>
      <th></th>
      <th>2aabd6</th>
      <th colspan=4>#29984 55e249</th>
    </tr>
    <tr>
      <th></th>
      <th>Total</th>
      <th>Total</th>
      <th>+/-</th>
      <th>New</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td><b>Rate</b></td>
      <td>85.26%</td>
      <td>85.26%</td>
      <td>-0.00%</td>
      <td>-</td>
    </tr>
    <tr>
      <td><b>Hits</b></td>
      <td>108581</td>
      <td>108580</td>
      <td>-1</td>
      <td>0</td>
    </tr>
    <tr>
      <td><b>Misses</b></td>
      <td>18769</td>
      <td>18770</td>
      <td>+1</td>
      <td>0</td>
    </tr>
  </tbody>
</table>

[Diff coverage report](https://mooseframework.inl.gov/docs/PRs/29984/coverage_diff/framework)

[Full coverage report](https://mooseframework.inl.gov/docs/PRs/29984/coverage/framework)


## Modules coverage

Coverage did not change

## Full coverage reports

<details>
<summary>Reports</summary>

- [`framework`](https://mooseframework.inl.gov/docs/PRs/29984/coverage/framework)
- [`chemical_reactions`](https://mooseframework.inl.gov/docs/PRs/29984/coverage/chemical_reactions)
- [`combined`](https://mooseframework.inl.gov/docs/PRs/29984/coverage/combined)
- [`contact`](https://mooseframework.inl.gov/docs/PRs/29984/coverage/contact)
- [`electromagnetics`](https://mooseframework.inl.gov/docs/PRs/29984/coverage/electromagnetics)
- [`external_petsc_solver`](https://mooseframework.inl.gov/docs/PRs/29984/coverage/external_petsc_solver)
- [`fluid_properties`](https://mooseframework.inl.gov/docs/PRs/29984/coverage/fluid_properties)
- [`fsi`](https://mooseframework.inl.gov/docs/PRs/29984/coverage/fsi)
- [`functional_expansion_tools`](https://mooseframework.inl.gov/docs/PRs/29984/coverage/functional_expansion_tools)
- [`geochemistry`](https://mooseframework.inl.gov/docs/PRs/29984/coverage/geochemistry)
- [`heat_transfer`](https://mooseframework.inl.gov/docs/PRs/29984/coverage/heat_transfer)
- [`level_set`](https://mooseframework.inl.gov/docs/PRs/29984/coverage/level_set)
- [`misc`](https://mooseframework.inl.gov/docs/PRs/29984/coverage/misc)
- [`navier_stokes`](https://mooseframework.inl.gov/docs/PRs/29984/coverage/navier_stokes)
- [`optimization`](https://mooseframework.inl.gov/docs/PRs/29984/coverage/optimization)
- [`peridynamics`](https://mooseframework.inl.gov/docs/PRs/29984/coverage/peridynamics)
- [`phase_field`](https://mooseframework.inl.gov/docs/PRs/29984/coverage/phase_field)
- [`porous_flow`](https://mooseframework.inl.gov/docs/PRs/29984/coverage/porous_flow)
- [`ray_tracing`](https://mooseframework.inl.gov/docs/PRs/29984/coverage/ray_tracing)
- [`rdg`](https://mooseframework.inl.gov/docs/PRs/29984/coverage/rdg)
- [`reactor`](https://mooseframework.inl.gov/docs/PRs/29984/coverage/reactor)
- [`richards`](https://mooseframework.inl.gov/docs/PRs/29984/coverage/richards)
- [`scalar_transport`](https://mooseframework.inl.gov/docs/PRs/29984/coverage/scalar_transport)
- [`solid_mechanics`](https://mooseframework.inl.gov/docs/PRs/29984/coverage/solid_mechanics)
- [`solid_properties`](https://mooseframework.inl.gov/docs/PRs/29984/coverage/solid_properties)
- [`stochastic_tools`](https://mooseframework.inl.gov/docs/PRs/29984/coverage/stochastic_tools)
- [`thermal_hydraulics`](https://mooseframework.inl.gov/docs/PRs/29984/coverage/thermal_hydraulics)
- [`xfem`](https://mooseframework.inl.gov/docs/PRs/29984/coverage/xfem)

</details>



This comment will be updated on new commits.

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/2769448/), step Versioner verify on 14a3651 wanted to post the following:

### Versioner templates

Found 16 templates, 0 failed


### Versioner versions

Found 10 packages, 1 changed, 0 failed

| package   | status   | hash    |   to hash | version            | to version         |
|-----------|----------|---------|-----------|--------------------|--------------------|
| seacas    | CHANGE   | 2dd352c |  6.7e+221 | 2024.08.15 build 2 | 2025.02.27 build 0 |

loganharbour reviewed (changes requested):
Will need to rebase due do https://github.com/idaholab/moose/pull/30042. otherwise, all good

cticenhour commented:
Yeah - I just wanted to see how the testing shook out, since there were some failures last time that I wasn't sure where they were coming from.

loganharbour commented:
> Yeah - I just wanted to see how the testing shook out, since there were some failures last time that I wasn't sure where they were coming from.

oh, sorry. wasn't a draft so I assumed you were good

milljm commented:
https://github.com/idaholab/moose/pull/29884 affects this PR as well. Depending on "who wins".

But because the above PR changes the entire stack, yours will most likely make it in before mine.

cticenhour commented:
@milljm Given the exact content of this, I feel that this PR can be merged absolutely whenever. I know you've been waiting on that .GOV PR for a bit now.

milljm reviewed (approved):
👍

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/2770074/), step Versioner verify on ad424b4 wanted to post the following:

### Versioner templates

Found 16 templates, 0 failed


### Versioner versions

Found 10 packages, 1 changed, 0 failed

| package   | status   | hash    |   to hash | version            | to version         |
|-----------|----------|---------|-----------|--------------------|--------------------|
| seacas    | CHANGE   | 2dd352c |  6.7e+221 | 2024.08.15 build 2 | 2025.02.27 build 0 |

cticenhour committed:
Update versioner hashes

This also adds a missing ticket reference to the most-recent libmesh update hash changes

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/2772829/), step Versioner verify on b58fac9 wanted to post the following:

### Versioner templates

Found 16 templates, 0 failed


### Versioner versions

Found 10 packages, 1 changed, 0 failed

| package   | status   | hash    |   to hash | version            | to version         |
|-----------|----------|---------|-----------|--------------------|--------------------|
| seacas    | CHANGE   | 2dd352c |  6.7e+221 | 2024.08.15 build 2 | 2025.02.27 build 0 |

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/2774855/), step Versioner verify on a0f0f33 wanted to post the following:

### Versioner templates

Found 16 templates, 0 failed


### Versioner versions

Found 10 packages, 1 changed, 0 failed

| package   | status   | hash    |   to hash | version            | to version         |
|-----------|----------|---------|-----------|--------------------|--------------------|
| seacas    | CHANGE   | 2dd352c |  6.7e+221 | 2024.08.15 build 2 | 2025.02.27 build 0 |

cticenhour commented:
@loganharbour this is ready to go!

milljm reviewed (approved):
looks good to me!

loganharbour reviewed (approved):
Good to go whenever you're ready
